service/whatsapp/whatsapp.go

```diff
@@ -19,7 +20,7 @@ const (
 
 var sessionFilePath = filepath.Join(os.TempDir(), "whatsappSession.gob")
 
-// whatsappClient abstracts go-whatsapp for writing unit tests
+// mockWhatsappClient abstracts go-whatsapp for writing unit tests
```

Comment:
Should be `whatsappClient` instead of `mockWhatsappClient `.

Reply:
Appreciate it! Fixed this for _Plivo_ service too.